editoast/src/views/projects.rs

```diff
@@ -86,21 +88,20 @@ struct ProjectCreateForm {
 impl From<ProjectCreateForm> for Project {
```

Comment:
```suggestion
impl From<ProjectCreateForm> for <Project as modelsv2::Model>::Changeset {
```

I'll make a PR to add a type to simplify this (like in https://github.com/rust-lang/rust/issues/38078#issuecomment-1672202401).